sql/src/test/java/org/apache/druid/sql/calcite/SqlTestFrameworkConfig.java

```diff
@@ -103,24 +106,63 @@ public SqlTestFrameworkConfig defaultConfig()
     public Statement apply(Statement base, Description description)
     {
       config = description.getAnnotation(SqlTestFrameworkConfig.class);
+      if (config == null) {
+        config = description.getTestClass().getAnnotation(SqlTestFrameworkConfig.class);
```

Comment:
are you planning to use class level configs somewhere in future?

Reply:
not a 100% sure how I want to have this in the end - for now I've removed the possibility to process class level annotations; I'll get back to that when that will be needed..